pkg/reader/reader.go

```diff
@@ -28,10 +28,10 @@ type FileReader struct {
 	StopFlag    bool
 }
 
-func New(fileIdx int, file *config.File, clientRequestChs []chan base.ClientRequest, errCh chan<- base.ErrData) (*FileReader, error) {
+func New(fileIdx int, comma rune, file *config.File, clientRequestChs []chan base.ClientRequest, errCh chan<- base.ErrData) (*FileReader, error) {
 	switch strings.ToLower(*file.Type) {
 	case "csv":
-		r := csv.CSVReader{CSVConfig: file.CSV}
+		r := csv.CSVReader{CSVConfig: file.CSV, Comma: comma}
```

Comment:
ditto

Reply:
you need think all csv file have one common format

Reply:
> you need think all csv file have one common format

Please refer the above replies.